content/structure.md

```diff
@@ -162,15 +196,24 @@ import '../imports/startup/server/security.js';
 import '../imports/api/api.js';
 ```
 
-You can see that here we don't actually import any variables from these files - we just import them so that they execute in this order.
+Our main server entry point `server/main.js` then imports this startup module. You can see that here we don't actually import any variables from these files - we just import them so that they execute in this order.
+
+<h3 id="importing-meteor-globals">Importing Meteor "pseudo-globals"</h3>
+
+For backwards compatibility Meteor 1.3 still provides Meteor's global namespacing for the Meteor core package as well as for other Meteor packages you include in your application. You can also still directly call functions such as [`Meteor.publish`](http://docs.meteor.com/#/full/meteor_publish), as in previous versions of Meteor, without first importing them. However, it is recommended best practice that you first load all the Meteor "pseudo-globals" using the `import { Name } from 'meteor/package'` syntax before using them. For instance:
+
+```js
+import { Meteor } from 'meteor/meteor';
+import { EJSON } from 'meteor/ejson';
+```
 
 <h2 id="load-order">Default file load order</h2>
 
-Even though it is recommended that you write your application to use ES2015 modules and the `imports/` directory, Meteor 1.3 continues to support eager loading of files, using these default load order rules, to provide backwards compatibility with applications written for Meteor 1.2 and earlier.
+Even though it is recommended that you write your application to use ES2015 modules and the `imports/` directory, Meteor 1.3 continues to support eager loading of files, using these default load order rules, to provide backwards compatibility with applications written for Meteor 1.2 and earlier. You may combine both eager loading and lazy loading using `import` in a single app. Any import statements are evaluated in the order they are listed in a file when that file is loaded and evaluated using these rules.
 
 There are several load order rules. They are *applied sequentially* to all applicable files in the application, in the priority given below:
 
-1. HTML template files are *always* loaded before everything else
+1. HTML template files are **always** loaded before everything else
 2. Files beginning with `main.` are loaded **last**
 3. Files inside **any** `lib/` directory are loaded next
```

Comment:
Yes, it can be a bit confusing, but the order listed is the priority order that the rules are evaluated, not files are loaded, as shown in the little example below this section. If you can come up with a better way to express this I'm sure MDG will accept the change.